app/models/concerns/digital_object_concerns/assets/validations.rb

```diff
@@ -5,5 +5,11 @@ module DigitalObjectConcerns::Assets::Validations
 
   included do
     validates :asset_type, inclusion: { in: Hyacinth::DigitalObject::AssetType::VALID_TYPES, message: "Invalid asset type: %{value}" }
+    validate :validate_rights_updates
+  end
+
+  def validate_rights_updates
+    return unless rights.keys.detect { |key| rights[key].present? && !'access_condition'.eql?(key.to_s) }
```

Comment:
I think i am calling the 'access_condition section `restriction_on_access` to follow what the field group was called in HY2. Which should we follow?

Reply:
I am spitballing based on what we had in master. Can we update the subsection key in your branch after merge?

Reply:
hmm, I wondering if this should wait until after I merge my code because i have new rights mutations in place.